src/gerbil/test/c3-test.ss

```diff
@@ -1,35 +1,94 @@
 ;; -*- Gerbil -*-
 ;;; © fare@tunes.org
 ;;;; Testing the c3 linearization algorithm
+prelude: "../prelude/core"
```

Comment:
uhm this is not ideal for running the tests after things are built; we should probably remove it before merge or comment it out for future use.